freeipa/src/main/java/com/sequenceiq/freeipa/entity/Stack.java

```diff
@@ -223,7 +223,7 @@ public void setGatewayport(Integer gatewayport) {
         this.gatewayport = gatewayport;
     }
 
-    public boolean isUseCcm() {
+    public Boolean getUseCcm() {
         return Boolean.TRUE.equals(useCcm);
     }
```

Comment:
This is not a pure getter and cannot ever return `null`, so perhaps the original signature was more appropriate. (But then a new method is needed for the real getter.)

Reply:
I don't wanna. this is a very rare edge case (2 occurrences) and would unnecessarily complicate the checker logic.